go/host/rpc_server.go

```diff
@@ -33,6 +33,10 @@ func NewRPCServer(config config.HostConfig, host *Node) RPCServer {
 		rpcConfig.WSOrigins = []string{allOrigins}
 	}
 
+	// allows calls from any vhost
+	// TODO review if this poses a security issue
+	rpcConfig.HTTPVirtualHosts = []string{"*"}
```

Comment:
@otherview Shouldn't this be in the `if config.HasClientRPCHTTP {` block above?

And you can use the `allOrigins` constant, instead of `"*"`.